package.json

```diff
@@ -20,6 +20,7 @@
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
     "hast-util-is-element": "^1.1.0",
+    "noir-cra": "github:noir-lang/noir-cra",
```

Comment:
Why do we need to import `noir-cra` to the docs?

Reply:
We don't, sorry I switched context with git stash and forgot I was trying to have code execution on another branch 🤦‍♂️